### PR TITLE
Adds a new set of cyberlimbs/IPC parts.

### PR DESCRIPTION
**Pull request by TravellingMerchant**
This PR adds a brand new set of cyberlimbs/IPC parts to the parts list for all races. Big thanks to KasparoVv who has cleaned up the accessories file to allow the monitor to work.

It comes with an alternate head and a monitor head. Base colour is red, but a recognisably different tone from security. (It's not like IPC's cant recolour themselves anyway.

shellguard limbs

This is a view of the new parts in a gif.
Here are some examples in game.

All the IPC accessories work fine with it.

It is based off my recently added Noble cyborgs as they are from my same fictional company.

### CODE REVIEW

**Comment by Spacemanspark** [comment]
Looks neato.
:+1:

File: icons/mob/human_races/cyberlimbs/shellguard/robolimbs.dm [hunk 1]
```diff
@@ -0,0 +1,141 @@
+var/global/list/all_robolimbs = list()
```
Comment (KasparoVy) [inline icons/mob/human_races/cyberlimbs/shellguard/robolimbs.dm]: This file shouldn't be necessary as the shellguard datums are defined in the above file  code/modules/surgery/organs/robolimbs.dm

**Comment by LightFire53** [comment]
Seems interesting, anyway. Don't know if I would personally use it, but more customization is always welcome.

**Comment by Anoonki** [comment]
I wasn't sure about these when you posted just that top screenshot there, but demonstrating what it actually looks like on a clothed person, they look really nice.

**Comment by DesolateG** [comment]
I like em

File: icons/mob/human_races/cyberlimbs/shellguard/robolimbs.dm (hunk 1 above)
Comment (TravellingMerchant) [inline icons/mob/human_races/cyberlimbs/shellguard/robolimbs.dm]: I see what happened. I dragged and dropped the wrong file. I wanted to replace the icons in my fork with a slightly better hand sprite.

**Comment by Fruerlund** [comment]
Very nice, more diverse characters always usefull, have you considered different colors?

**Comment by TravellingMerchant** [comment]
@Fruerlund IPC's can recolour themselves.

**Comment by granodd** [comment]
Very neat indeed. I like the design of the frame. The only concern I have is how the frame's color will interact with being changed, as I know that some of the part options for IPCs have a "default color" that looks rather hideous when you try to slap other colors onto it. Even if red is the default color, though, it looks mighty fine.

**Comment by fludd12** [comment]
I'd prefer for the sprite to be greyscaled for better color options, but otherwise it looks very nice.

**Comment by DesolateG** [comment]
@fludd12 the colors can be changed

**Comment by AffectedArc07** [comment]
I think these would look quite good.

**Pull request merged**